micro_http/src/common/mod.rs

```diff
@@ -99,7 +110,7 @@ impl Method {
 /// let version = Version::try_from(b"HTTP/1.1");
 /// assert!(version.is_ok());
 ///
-/// let version = Version::try_from(b"http/1.1");
+/// let version = Version::try_from(b"http/2.0");
```

Comment:
Why is this changed? `http/1.1` should be invalid because the protocol name should be in uppercase.